packages/http/package.json

```diff
@@ -32,6 +32,7 @@
     "pino": "^5.13.2",
     "tslib": "^1.10.0",
     "type-is": "^1.6.18",
+    "uri-templates": "^0.2.0",
```

Comment:
Can we use the same library everywhere? URIJS is going away with proxy command, but the example generation is using another library (see the CLI). It'd be cool to replace it and use the same here as well.

Reply:
My bad that I did not unified those libraries.

I looked on [uri-templates](https://github.com/geraintluff/uri-templates#readme) and [uri-template-lite](https://github.com/litejs/uri-template-lite#readme). Both seem to support full spec. The latter is 3x smaller and the code looks less dense. My vote goes on `-lite` variant.

Now, cli and http uses `uri-template-lite`.

Reply:
Saturday task: create typings for such library :trollface: